.prefs/ubuntu.sh

```diff
@@ -122,7 +122,7 @@ sudo sysctl -p --system
 sudo apt-get install lib64stdc++6:i386 mesa-utils
 sudo apt-get install mesa-utils
 
-cd $ANDROID_HOME/emulator/lib64
+cd "$ANDROID_HOME/emulator/lib64" || exit
```

Comment:
Why exit tho?

Reply:
This used to be flagged but not anymore. Weird. Will remove.